pandas/src/period.pyx

```diff
@@ -683,7 +683,6 @@ cdef class _Period(object):
 
         if isinstance(freq, compat.string_types):
             freq = freq.upper()
```

Comment:
can we remove ``.upper`` also, because it is handled in ``get_offset`` (called from ``to_offset``)?

- https://github.com/pydata/pandas/blob/1e1e9b348bb3f256c2b4997db090a1d35da9938b/pandas/tseries/frequencies.py#L493